src/blurredLocation.js

```diff
@@ -62,6 +61,8 @@ BlurredLocation = function BlurredLocation(options) {
     });
     var geometry = Blurred.responseJSON.results[0].geometry.location;
     options.map.setView([geometry.lat, geometry.lng],options.zoom);
+    $("#lat").val(geometry.lat);
```

Comment:
Sure, and one can input onComplete function in options and this will be the default